### PR TITLE
test: add e2e test for universal compiler

### PR DESCRIPTION
**Pull request by snitin315**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [x] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
It is test

### Motivation / Use-Case

migrate universal compiler test to e2e testing.

### Breaking Changes
none

### Additional Info
No

### CODE REVIEW

File: test/e2e/universal-compiler.test.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    await browser.close();
+    await server.close();
+  });
+
+  it("server bundle should NOT have the inlined the client runtime", async () => {
+    // we wouldn't normally request a server bundle
+    // but we'll do it here to check the contents
+    const compiler = webpack(config);
+    const devServerOptions = {
+      host: "127.0.0.1",
```
Comment (alexander-akait) [inline test/e2e/universal-compiler.test.js:61]: Let's avoid using `host`, it is unnecessary (same for other test)

**Review by alexander-akait — APPROVED** [review]
Big thanks

**Comment by snitin315** [comment]
Looks like a random macOs faliure, can we ignore it?

**Comment by alexander-akait** [comment]
Yep, I will improve our test for more stable

**Pull request merged**